charts/invenio/templates/uwsgi-configmap.yaml

```
@@ -55,3 +55,7 @@ data:
     die-on-term = true
     # hook-master-start = unix_signal:2 gracefully_kill_them_all
     # hook-master-start = unix_signal:15 gracefully_kill_them_all
+    {{- range $key, $val := .Values.invenio.uwsgi_extra_config }}
+    {{ $key }} = {{ $val }}
+    {{- end -}}
```

Maybe we drop the trailing `-`, just in case we want to add something more after this line in the future…

```suggestion
    {{- end }}
```